ai-platform-unified/tutorials/unofficial/pytorch-image-classification-custom-train-container-multi-replica/trainer/task.py

```diff
@@ -0,0 +1,255 @@
+"""Main program for PyTorch distributed training.
```

Comment:
I think this should have

```
# Copyright 2021 Google LLC
#
# Licensed under the Apache License, Version 2.0 (the "License");
```

etc.

added as a header?

(Keeping the "Adapted from" line that you already have, as well)